lib/workflow-executor.ts

```
@@ -648,13 +649,340 @@ export class WorkflowExecutor {
 
+
         // Also store the results in the node data for UI access
         node.data.results = updatedElements;
         break;
 
+      case "materialNode":
+        console.log("Processing materialNode", { node, inputValues });
+
+        // Extract elements from input
+        let materialElements = [];
+        if (Array.isArray(inputValues.input)) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)